runner/jobserv_runner/handlers/simple.py

```diff
@@ -230,7 +230,7 @@ def hung_cb():
                         stderr=f,
                     )
 
-            cmd = ["docker", "run"]
+            cmd = ["docker", "run", "--rm"]
```

Comment:
with this, I think we can remove the `docker_clean` logic

Also -  the commit message should explain the *why* and subject can drop the `improvement` line.

Reply:
Improved the commit message and the title. I think we can remove the `docker_clean` logic too, as what I have seen is that the other directories inside `docker` did not contribute much to the disk space used.